presto-spi/src/main/java/com/facebook/presto/spi/SplitContext.java

```diff
@@ -22,6 +25,7 @@
 import static java.util.Objects.requireNonNull;
 
 // TODO: Use builder pattern for SplitContext if we are to add optional field
+@ThriftStruct
```

Comment:
Hmm, thrift annotations are already in the SPI ... Any thoughts on whether we should consider annotating the `HiveSplit` or other objects and go the handle resolves way?